backend/tests/test_project.py

```diff
@@ -1,7 +1,7 @@
 import json
 import unittest
 import uuid
-from flask import current_app
+from flask import current_app, request
```

Comment:
@peterghrong Is there a reason why this keeps on popping up?

Reply:
Might have pulled from a bad branch lemme just clean everything up